docker-compose.yml

```diff
@@ -119,6 +119,7 @@ services:
       - --map-url=map-server:8090
       - --tls-key=/kt/server.key
       - --tls-cert=/kt/server.crt
+      - --auth-type=insecure-fake
```

Comment:
Should there be a scary warning somewhere?

Reply:
Not particularly as this is the deployment for testing

Reply:
Not sure people who check this code out will know that.